web/static/slack/connect/index.html

```diff
@@ -125,10 +127,68 @@ <h1>Slack - Create a New Connection</h1>
         Click the button below to install and authorize<br />
         the <b>AutoKitteh</b> app in your Slack workspace
       </p>
-      <!-- TODO(ENG-799): remove ID once its unused -->
-      <a id="oauth-link" href="/oauth/start/slack" class="clickable"
-        >Start OAuth Flow</a
-      >
+      <a id="oauth-link" href="/oauth/start/slack" class="clickable">Start OAuth Flow</a>
+    </div>
+
+    <div class="tab" id="tab3">
+      <p class="info">Information:</p>
+      <ul class="links">
+        <li>
+          <a
+            href="https://docs.autokitteh.com/integrations/slack/custom-oauth"
+            target="_blank"
+            class="info"
+          >
+            AutoKitteh guide: configuring a custom Slack app
+          </a>
+        </li>
+        <li>
+          <a
+            href="https://api.slack.com/authentication/oauth-v2"
+            target="_blank"
+            class="info"
+          >
+            Background about installing Slack apps with OAuth
+          </a>
+        </li>
+      </ul>
+
+      <form id="save-form-custom-oauth" method="post" action="/slack/custom-oauth">
+        <div class="form-group">
+          <label for="client_id">Client ID</label>
+          <input
+            type="text"
+            name="client_id"
+            autocomplete="off"
+            spellcheck="false"
+            required
+          />
+        </div>
+
+        <div class="form-group">
+          <label for="client_secret">Client Secret</label>
+          <input
+            type="text"
+            name="client_secret"
+            autocomplete="off"
+            spellcheck="false"
+            required
+          />
+        </div>
+
+        <div class="form-group">
+          <label for="signing_secret">Signing Secret</label>
+          <input
+            type="text"
+            name="signing_secret"
+            autocomplete="off"
+            spellcheck="false"
+            required
+          />
+        </div>
+
+        <button type="submit" class="submit-btn">Start Custom OAuth Flow</button>
```

Comment:
I see the details that the user is supposed to provide to us, but what about the webhook slug that we provide to them to accept events? Or is that a separate phase?

Reply:
I don't understand. What webhook slug? Are you referring to what replaced our http connections? That webhook?
i.e.:
```
  triggers:
    - name: cross_repo
      type: webhook
      event_type: get
      call: program.py:on_cross_repo
```

Reply:
Oh, my mistake - I kept thinking about having a unique OAuth response webhook, but I guess it's redundant in the way you implemented it (which is good).